Sources/MapboxMaps/Style/ModelLayer.swift

```diff
@@ -1,96 +0,0 @@
-// swiftlint:disable nesting
-
-import Foundation
-import MapboxCoreMaps
-import MapboxCommon
-
-/// - SeeAlso: [Online documentation](https://www.mapbox.com/mapbox-gl-style-spec/#layers-model)
-internal struct ModelLayer: Layer {
-
-    // MARK: - Conformance to `Layer` protocol
-    internal var id: String
-    internal var type: LayerType
-    internal var filter: Expression?
-    internal var source: String?
-    internal var sourceLayer: String?
-    internal var minZoom: Double?
-    internal var maxZoom: Double?
-
-    /// Layer layout properties are applied early in the rendering process and define how data for that layer is passed to the GPU.
-    /// Changes to a layout property require an asynchronous "layout" step.
-    internal var layout: ModelLayer.Layout?
-
-    internal struct Layout: Codable {
-
-        /// Whether this layer is displayed.
-        internal var visibility: Value<Visibility>?
-
-        internal init() {
-            self.visibility = .constant(.visible)
-        }
-
-        enum CodingKeys: String, CodingKey {
-            case visibility
-        }
-    }
-
-    /// Changes to a paint property are cheap and happen synchronously.
-    internal var paint: ModelLayer.Paint?
-
-    internal struct Paint: Codable {
```

Comment:
seems the layer properties are not grouped into paint and layout properties any more, I guess this is generated the same way with other layers?

Reply:
Yeah, generated layer properties are unrolled into a flat list